### PR TITLE
Fix sample issues

### PR DESCRIPTION
**Pull request by anupama-pathirage**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/http-compression/http_compression.server.out [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# To run the service, put the code in a `http_compression.bal` file
-# and execute the following command from `$BALLERINA_HOME/bin`.
+# To start the service, navigate to the directory that contains the
```
Comment (samgnaniah) [inline examples/http-compression/http_compression.server.out]:
```suggestion
# To start the services, navigate to the directory that contains the
```

File: examples/http-circuit-breaker/http_circuit_breaker.server.out [hunk 2]
```diff
@@ -1,5 +1,5 @@
-#At the command line, navigate to the directory that contains the 
-#`http_circuit_breaker.bal` file and run the `ballerina run` command to start the server.
+# To start the service, navigate to the directory that contains the
```
Comment (samgnaniah) [inline examples/http-circuit-breaker/http_circuit_breaker.server.out]:
```suggestion
# To start the services, navigate to the directory that contains the
```

File: examples/http-disable-chunking/http_disable_chunking.server.out [hunk 3]
```diff
@@ -1,5 +1,5 @@
-# To run the service, put the code in a `http_disable_chunking.bal` file
-# and execute the following command from `$BALLERINA_HOME/bin`.
+# To start the service, navigate to the directory that contains the
```
Comment (samgnaniah) [inline examples/http-disable-chunking/http_disable_chunking.server.out]:
```suggestion
# To start the services, navigate to the directory that contains the
```

File: examples/http-failover/http_failover.server.out [hunk 4]
```diff
@@ -1,5 +1,5 @@
-# To run the service, put the code in `http_failover.bal`
-# and execute the following command from `$BALLERINA_HOME/bin`.
+# To start the service, navigate to the directory that contains the
```
Comment (samgnaniah) [inline examples/http-failover/http_failover.server.out]:
```suggestion
# To start the services, navigate to the directory that contains the
```

File: examples/http-load-balancer/http_load_balancer.server.out [hunk 5]
```diff
@@ -1,5 +1,5 @@
-# To run the service, put the code in `http_load_balancer.bal`
-# and execute the following command from `$BALLERINA_HOME/bin`.
+# To start the service, navigate to the directory that contains the
```
Comment (samgnaniah) [inline examples/http-load-balancer/http_load_balancer.server.out]:
```suggestion
# To start the services, navigate to the directory that contains the
```

File: examples/http-retry/http_retry.server.out [hunk 6]
```diff
@@ -1,5 +1,5 @@
-#At the command line, navigate to the directory that contains the 
-#`http_retry.bal` file and run the `ballerina run` command to start the server.
+# To start the service, navigate to the directory that contains the
```
Comment (samgnaniah) [inline examples/http-retry/http_retry.server.out]:
```suggestion
# To start the services, navigate to the directory that contains the
```

File: examples/http-timeout/http_timeout.server.out [hunk 7]
```diff
@@ -1,5 +1,5 @@
-#At the command line, navigate to the directory that contains the
-#`http_timeout.bal` file and run the `ballerina run` command to start the server.
+# To start the service, navigate to the directory that contains the
```
Comment (samgnaniah) [inline examples/http-timeout/http_timeout.server.out]:
```suggestion
# To start the services, navigate to the directory that contains the
```

File: examples/any-type/any_type.out [hunk 8]
```diff
@@ -1,3 +1,5 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file and issue the `ballerina run` command.
 $ ballerina run any_type.bal
 25
 Person p's age: 25
```
Comment (MaryamZi) [inline examples/any-type/any_type.out]: needs a new line.

File: examples/check/check.out [hunk 9]
```diff
@@ -1,5 +1,5 @@
-#At the command line, navigate to the directory that contains the 
-#`.bal` file and run the `ballerina run` command. 
+# To run this sample, navigate to the directory that contains the
+# `.bal` file and issue the `ballerina run` command.
 $ ballerina run check.bal
 12
 'string' cannot be converted to 'int' {}
```
Comment (MaryamZi) [inline examples/check/check.out]: needs a new line.

File: examples/counter-metrics/counter_metrics.bal [hunk 10]
```diff
@@ -24,8 +25,9 @@ service onlineStoreService on new http:Listener(9090) {
         localCounter.increment(amount = 20);
 
         //Create a counter with optional fields description, and tags.
-        observe:Counter registeredCounter = new("product_total_product_order_quantity",
-            desc = "Total quantity required", tags = {prodName:"HeadPhone", prodType:"Electronics"});
+        observe:Counter registeredCounter = new("total_product_order_quantity",
+            desc = "Total quantity required",
+            tags = {prodName:"HeadPhone", prodType:"Electronics"});
```
Comment (MaryamZi) [inline examples/counter-metrics/counter_metrics.bal]: shall we fix formatting here with spaces?

File: examples/http-failover/http_failover.bal [hunk 11]
```diff
@@ -89,7 +90,8 @@ service mock on backendEP {
     resource function mockResource(http:Caller caller, http:Request req) {
         var result = caller->respond("Mock Resource is Invoked.");
         if (result is error) {
-           log:printError("Error sending response from mock service", err = result);
+           log:printError("Error sending response from mock service",
+                            err = result);
```
Comment (MaryamZi) [inline examples/http-failover/http_failover.bal]: shall we align this similar to L76?

File: examples/http-filters/http_filters.bal [hunk 12]
```diff
@@ -5,21 +5,25 @@ final string filter_name_header = "X-filterName";
...
+                        http:FilterContext context)
                         returns boolean {
         // set a header for filter
         request.setHeader(filter_name_header, filter_name_header_value);
         return true;
     }
 
-    public function filterResponse(http:Response response, http:FilterContext context) returns boolean {
+    public function filterResponse(http:Response response,
+                                    http:FilterContext context)
```
Comment (MaryamZi) [inline examples/http-filters/http_filters.bal]: inconsistent alignment here and with L9?

**Pull request merged**